### PR TITLE
[fix]: [ci-9132]: ubi image upgrades

### PR DESCRIPTION
**Pull request by abhay084**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/52592)

### CODE REVIEW

File: 332-ci-manager/build/container/Dockerfile-ci-manager-cie-jdk [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # To be used when building in CIE
 
-FROM us.gcr.io/platform-205701/ubi/ubi-java:8.8 as base
+FROM us.gcr.io/platform-205701/harness/ubi8:8.8 as base
```
Comment (raghavharness) [inline 332-ci-manager/build/container/Dockerfile-ci-manager-cie-jdk:8]: Change in `332-ci-manager/build/container/Dockerfile-ci-manager-multistage` as well

**Comment by abhay084** [comment]
trigger uts

File: product/ci/scm/docker/Dockerfile [hunk 2]
```diff
@@ -18,7 +18,7 @@ RUN GRPC_HEALTH_PROBE_VERSION=v0.4.19 && \
     chmod +x /bin/grpc_health_probe
 
 # Second stage
-FROM us.gcr.io/platform-205701/ubi/ubi-go:8.8
+FROM FROM us.gcr.io/platform-205701/harness/ubi8/go1:1.20.5
```
Comment (raghavharness) [inline product/ci/scm/docker/Dockerfile:21]: Using 2 FROM ? typo ?

File: product/log-service/docker/Dockerfile [hunk 3]
```diff
@@ -12,7 +12,7 @@
 #
 # > docker build -t harness/log-service:valpha-0.1 -f product/log-service/docker/Dockerfile $(bazel info bazel-bin)/product/log-service/
 
-FROM us.gcr.io/platform-205701/ubi/ubi-go:8.8
+FROM FROM us.gcr.io/platform-205701/harness/ubi8/go1:1.20.5
```
Comment (raghavharness) [inline product/log-service/docker/Dockerfile:15]: ditto

File: product/log-service/docker/Dockerfile-ubi [hunk 4]
```diff
@@ -10,7 +10,7 @@
 #
 # > docker build -t harness/log-service:valpha-0.1 -f product/log-service/docker/Dockerfile-ubi $(bazel info bazel-bin)/product/log-service/
 
-FROM us.gcr.io/platform-205701/ubi/ubi-base:8.8
+FROM FROM us.gcr.io/platform-205701/harness/ubi8:8.8
```
Comment (raghavharness) [inline product/log-service/docker/Dockerfile-ubi:13]: ditto

File: product/log-service/docker/Dockerfile (hunk 3 above)
Comment (raghavharness) [inline product/log-service/docker/Dockerfile:15]: ditto

File: product/log-service/docker/Dockerfile-ubi (hunk 4 above)
Comment (raghavharness) [inline product/log-service/docker/Dockerfile-ubi:13]: ditto

File: product/ci/scm/docker/Dockerfile (hunk 2 above)
Comment (raghavharness) [inline product/ci/scm/docker/Dockerfile:21]: Using 2 FROM ? typo ?

**Review by smjt-h — APPROVED** [review]
> Reviewed 1 of 4 files at r1, 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/52592)* status: :shipit: complete! ✓

**Review by raghavharness — APPROVED** [review]

**Pull request closed without merging**